### PR TITLE
Add set tle source command.

### PR DESCRIPTION
**Pull request by danada**
Having some trouble generating docs (I think it's a conflicting GO configuration from another project).

### CODE REVIEW

File: pkg/satellite/tle/set_tle_source.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer conn.Close()
+
+	client := stellarstation.NewStellarStationServiceClient(conn)
+
+	var source stellarstation.SetTleSourceRequest_Source
+
+	if o.Source == "MANUAL" {
```
Comment (hoshir) [inline pkg/satellite/tle/set_tle_source.go]: Nit: You can use `strings.ToLower(o.Source)  == "manual"` to ignore case.

**Review by hoshir — COMMENTED** [review]
Could you update the document as well?

**Review by hoshir — APPROVED** [review]

File: go.mod [hunk 2]
```diff
@@ -18,3 +18,5 @@ require (
 	google.golang.org/grpc v1.21.1
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
 )
+
+go 1.13
```
Comment (danada) [inline go.mod:22]: @hoshir is this okay? It keeps getting added (probably because I'm using 1.13 on my machine).

Comment (hoshir) [reply go.mod:22]: Developers will need go version equal or above 1.13, but I think it's OK.

Comment (danada) [reply go.mod:22]: Okay, I've updated the main README to say 1.13 is required instead of 1.11beta2.

**Pull request merged**